docs/add-a-system-config.rst

```diff
@@ -3,14 +3,60 @@ Adding a System Configuration
 =============================
 
 ``benchpark/configs`` contains a directory for each system specified in Benchpark.
-If the software stack on your system is unlike the available configurations, 
+If your system is unlike the available configurations,
 you can add a new directory with a name which identifies the system.
 
-The following is required for each given system ``benchpark/configs/${SYSTEM}``:
+The naming convention for the systems is as following::
 
-1. ``spack.yaml`` defines default compiler and package names Spack should
+  Integrator-(opt)ClusterType-CPU-(opt)GPU-Network
+
+Benchpark has definitions for the following systems:
+- AWS-Hpc7a-EPYC4-EFA
+- HPECray-EPYC3-MI250X-Slingshot	(Frontier, Lumi, Tioga)
+- IBM-POWER9-V100-Infiniband	        (Sierra, Summit)
+- Penguin-XeonPlatinum-OmniPath
+- x86_64                                (generic x86 CPU only platform)
+
+The following files are required for each system ``benchpark/configs/${SYSTEM}``:
+
+1. ``system_definition.yaml``
```

Comment:
I think we should add a little bit more description here about what it is and how to fill it out